src/main/java/svc/data/citations/filters/CitationDateFilter.java

```diff
@@ -0,0 +1,25 @@
+package svc.data.citations.filters;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.ListIterator;
+
+import svc.data.citations.datasources.CITATION_DATASOURCE;
+import svc.models.Citation;
+
+public class CitationDateFilter {
+
+	public static List<Citation> FilterDates(List<Citation> citations){
```

Comment:
This method should probably just be called 'Filter' or 'FilterCitations' since that's what it does (it doesn't filter dates). It could also be called 'FilterByDate' or more specifically indicate what dates it is filtering out, like 'FilterByPastDates' or something.